### ISSUE TITLE
Sunset of BinTray

### ISSUE THREAD

**Issue by marcusdroy** [michaelklishin/quartz-mongodb#213]
As per the deprecation of bin tray hosting artifacts 

https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

It would appear this library will become unavailable soon, can it migrated to maven central? 

Thanks

**Comment by jim-olsen** [comment]
Yes, in fact now we are already seeing 'brownouts' where it is not pulling from bintray as of today...  So plus one on moving it to somewhere else that will continue to be supported

**Comment by bulivlad** [comment]
this is a real issue now as bintray became unavailable. we are seeing our builds fail as it is no longer able to pull the dependencies. any plan to move it to another repo soon?

**Comment by nikolap** [comment]
Agreed, would be great to have this mirrored elsewhere

**Comment by iori57** [comment]
Is it available in JCenter? our build is failing now too because bintray is gone

**Comment by bulivlad** [comment]
the easiest way to fix this was to fork the project and use github packages to store the artifacts 
see here https://github.com/bulivlad/quartz-mongodb/blob/da6555d4fe1b3b52221c402415f4b302d097b5e5/build.gradle#L104

**Comment by antpaw** [comment]
I really hope a more permanent solution can be worked out here.
I was able to fetch it form jitpack.io

```
	<repositories>
		<repository>
		    <id>jitpack.io</id>
		    <url>https://jitpack.io</url>
		</repository>
	</repositories>


	<dependency>
	    <groupId>com.github.michaelklishin</groupId>
	    <artifactId>quartz-mongodb</artifactId>
	    <version>2.1.0</version>
	</dependency>
```

you might want to fork and login to jitpack
